test/utils/ports.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+var ServicePorts = map[string]string{
+	// platform services
+	"support-scheduler": "59861",
+	// app services
+	"app-rfid-llrp-inventory":  "59711",
+	"app-service-configurable": "59701",
+	// device services
+	"device-gpio":            "59910",
+	"device-modbus":          "59901",
+	"device-mqtt":            "59982",
+	"device-onvif-camera":    "59984",
+	"device-rest":            "59986",
+	"device-rfid-llrp":       "59989",
+	"device-snmp":            "59993",
+	"device-usb-camera":      "59983",
+	"device-usb-camera/rtsp": "8554",
+	"device-virtual":         "59900",
+	// others
+	"ekuiper":          "20498",
+	"ekuiper/rest-api": "59720",
+	"ui":               "4000",
+}
```

Comment:
I separate map isn't necessary. You can use the [ServicePort](https://github.com/canonical/edgex-snap-testing/blob/8502c258ed65844ec255e0a85b9be783c2e6b6fa/test/utils/net.go#L50) function which does a inverse lookup of the port by service name.